### PR TITLE
Fix tests on Windows OS

### PR DESCRIPTION
**Pull request by jerone**
This PR contains the following:
* Fix (almost all) tests on Windows.

<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td>

image
	<td>

image

</table>

As you can see, one test is still failing. At some point I got it working and I have a feeling it's related to line-endings. Will investigate after this is merged, to get some flow going.

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+--ignore-engines true
```
Comment (ryu1kn) [inline .yarnrc]: Looks like we don't need to do this as Yarn 2 skips engine validation 👍

https://github.com/yarnpkg/yarn/issues/8072#issuecomment-753670472

**Review by ryu1kn — APPROVED** [review]
If you're okay with taking out the warning suppression settings, I'm happy for you to merge this.

I'm approving this PR anyway 👍

**Comment by ryu1kn** [comment]
Thanks for fixing the tests for Windows!

File: .yarnrc (hunk 1 above)
Comment (jerone) [reply .yarnrc]: I removed the file. 

I'm not super familiar with Yarn (yet), but it looks like we need to migrate this project over to yarn 2: https://yarnpkg.com/getting-started/migration

**Pull request merged**